api/swagger.yml

```diff
@@ -320,6 +320,10 @@ components:
         size_bytes:
           type: integer
           format: int64
+          description: |
+            The number of bytes in the object.  When lakeFS always populates
+            this field when returning ObjectStats.  This field is optional
```

Comment:
Should the second sentence just be "LakeFS always populates this field when returning ObjectStats"?

Reply:
Thanks!  Sometimes words are difficult :-/